ui-admin/src/study/notifications/NotificationConfigView.tsx

```diff
@@ -114,15 +127,32 @@ export default function NotificationConfigView({ studyEnvContext, portalContext
         <button type="button" className="btn btn-primary" onClick={saveConfig}>Save</button>
         <button type="button" className="btn btn-secondary ms-4"
           onClick={() => setShowSendModal(true)}>Send test email</button>
+        <button type="button" className="btn btn-danger ms-4" onClick={() => setShowDeleteModal(true)}>Delete</button>
       </div>
       {showSendModal && <TestEmailSender portalShortcode={portal.shortcode} environmentName={currentEnv.environmentName}
         onDismiss={() => setShowSendModal(false)} notificationConfig={workingConfig}/> }
-    </form> }
+      {showDeleteModal && (
+        <Modal show className="modal-lg" onHide={() => setShowDeleteModal(false)}>
```

Comment:
Style nit: I would probably just make this a regular `modal` since there isn't much content to display
```suggestion
        <Modal show className="modal" onHide={() => setShowDeleteModal(false)}>
```